### ISSUE TITLE
Cannot parse java version

### ISSUE THREAD

**Issue by mudlee** [beryx/badass-jlink-plugin#183]
I use this plugin in a gradle project where I have 2 subprojects. 

I get this error, but not always, just randomly:

```bash
> Task :sandbox:prepareMergedJarsDir FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sandbox:prepareMergedJarsDir'.
> Cannot parse java version: 

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
5 actionable tasks: 1 executed, 4 up-to-date
```

If I run this for few times, some of them fail, some not:

```bash
./gradlew clean && ./gradlew jlink
```

I tried to put the version to loads of places, but still it sometimes fail.

**Comment by mudlee** [comment]
Update: this should be a race condition or something like that, because if I run the jlink command with `--info` the problem doesn't occur.

**Comment by siordache** [comment]
I think that I found the cause of the race condition and I fixed it in 2.23.8. Give it a try!

**Comment by mudlee** [comment]
Looks like now it's working! Thanks for the really quick fix :)
